stdlib/2and3/genericpath.pyi

```diff
@@ -7,16 +7,21 @@ else:
     def commonprefix(m: Sequence[AnyStr]) -> AnyStr: ...
 if sys.version_info >= (3, 6):
     from os import PathLike
-    def exists(path: Union[AnyStr, PathLike[AnyStr]]) -> bool: ...
+    def exists(path: Union[AnyStr, PathLike[AnyStr], int]) -> bool: ...
```

Comment:
`AnyStr` actually isn't the best solution here because the typevar doesn't affect the return type. A Union would be better.